pkgs/development/libraries/libproxy/default.nix

```diff
@@ -28,6 +30,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-pqj1LwRdOK2CUu3hYIsogQIXxWzShDuKEbDTbtWkgnQ=";
   };
 
+  patches = lib.optionals stdenv.isDarwin [
```

Comment:
Like I said on the other PR, I do not think this is a good solution.

We should pass `-DWITH_DUKTAPE=OFF` to `cmakeFlags` on Darwin until libproxy does that by default.

Reply:
Right, but the way the build system is currently written, it does not know that we do not want to use duktape, thus it tries to request it using pkg-config, which is not registered.